From 0dc3c8a958b6bb2557c1a5cccbb0b658d7744099 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:26:35 +0200
Subject: [PATCH] edit formatting: use tabs instead of spaces

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 calculator.cpp | 6 +++++-
 calculator.h   | 6 +++---
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index d93e35b..e574624 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,10 +2,14 @@
 
 int Calculator::Add (double a, double b)
 {
+<<<<<<< HEAD
 	return a + b + 0.5;
+=======
+	return a + b;
+>>>>>>> cbcb06d (formatting: use tabs instead of spaces)
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.34.1

